deephyper/skopt/optimizer/optimizer.py

```diff
@@ -827,6 +859,9 @@ def _tell(self, x, y, fit=True):
             # handle failures
             yi = self._filter_failures(self.yi)
 
+            # convert multiple objectives to single scalar
+            yi = self._moo_scalarize(yi)
```

Comment:
is there a way to check the shape of `yi` to apply moo `if np.shape(yi)[1] > 1` then we do moo otherwise we stay classic?
```python
if np.shape(yi)[1] > 1:
  yi = self._moo_scalarize(yi)
```

Reply:
No worries. I can add that easily